contrib/new_post.py

```diff
@@ -0,0 +1,183 @@
+#!/usr/bin/env python3
```

Comment:
To be complete, could add a little version check? Need at least 3.7, or 3.8 if we're using the walrus operator as suggested in other comments:


```python
import sys

MIN_PYTHON = (3, 8)

if sys.version_info < min_version:
    print(f"Error: This script requires Python {'.'.join(map(str, MIN_PYTHON))} or higher.")
    sys.exit(1)
```

Reply:
Good idea. Python 3.7 is now EOL so I've made the min version 3.8.